src/Query.php

```diff
@@ -226,7 +227,7 @@ public function findLast($columns = array('*'))
 	 */
 	public function whereKey($id)
 	{
-		$this->query->where($this->model->getPrimaryKey() . ' = ' . $id);
```

Comment:
What's the setLimit here for? Feels less intitutive

Reply:
The primary key should be unique anyway right? So the setLimit is not really a necessity?

Reply:
Correct